doc/new-provider.md

```diff
@@ -0,0 +1,87 @@
+# How to add a new provider
+
+Adding a new Teller provider is very easy, but you still need to know where to start. we summarize the steps very shortly to make your life easier  
+
+
+## Provider implementation
+
+1. Copy the file [example.go](pkg/providers/example.go) from path [pkg/providers/example.go](../pkg/providers/example.go) and replace all the `example` places with your provider name. make sure the uppercase.
```

Comment:
It would be good if you could give some words about what's going on in this example file in a technical manner, so people would get to know the code better before they replacing.
Maybe mention that there is an interface called `Provider` that every provider must implement, and this file is like a skeleton with stubs implementations for the interface methods.